workspaces/cms-scripts/src/sitemap.ts

```diff
@@ -74,11 +91,31 @@ const parsePageDirectory = async (dir: string, depth = 0) => {
         const { hidden_page, link } = JSON.parse(page);
 
         if (!hidden_page && link) {
-          sitemapUrls.push({
+          let sitemapEntry: SitemapUrl = {
             url: link,
             changefreq: changefreqByDepth[depth],
             priority: priorityByDepth[depth]
-          });
+          }
+
+          if (link === '/en/learn/what-is-starknet') {
+            let videos: Video[] = [];
```

Comment:
Whenever possible avoid foreach
```suggestion
            const videos = playlist.map(chapter => ({
              thumbnail: `${domain}${chapter?.thumbnail}`,
              title: chapter?.title,
              description: chapter?.description,
              url: `${import.meta.env.VITE_CF_STREAM_URL}/${chapter?.videoId}/watch`
            }));
```